siriuspy/siriuspy/sofb/correctors.py

```diff
@@ -0,0 +1,573 @@
+"""Module to deal with correctors."""
+
+import time as _time
+import logging as _log
+import numpy as _np
+from .. import util as _util
```

Comment:
I am adopting the standard of separating groups of import by empty line. groups are: 1) default python packages, 2) sirius-developed packages other than current one, 3) imports ..., 4) imports .., and 5) imports .

Reply:
Ok, I agree. Can I fix this in next PR. I'm already preparing one.